### PR TITLE
Use cookies import

### PR DESCRIPTION
**Pull request by oogali**
PR to use the upstream cookies.js library rather than repeating code via copy-and-paste.

### CODE REVIEW

**Comment by oogali** [comment]
The unit tests I was referring to are in the upstream project:
* https://github.com/pablo-mayrgundter/cookies.js

File: package.json [hunk 1]
```diff
@@ -72,7 +72,7 @@
       "^.+\\.svg$": "<rootDir>/svgTransform.js"
     },
     "transformIgnorePatterns": [
-      "/node_modules/(?!(web-ifc|web-ifc-three|web-ifc-viewer|three)/)"
+      "/node_modules/(?!(@pablo-mayrgundter\/cookies.js|web-ifc|web-ifc-three|web-ifc-viewer|three)/)"
```
Comment (pablo-mayrgundter) [inline package.json:75]: Why is this needed?

File: src/privacy/Privacy.js [hunk 2]
```diff
@@ -1,7 +1,8 @@
 import {
   getCookie as getCookiePrivate,
-  getCookieBoolean as getCookieBooleanPrivate,
```
Comment (pablo-mayrgundter) [inline src/privacy/Privacy.js:3]: I moved these over to the library and checked.. yeah, the getCookieBoolean has the type check (as well as the value check), so I think we should use.

File: src/privacy/Privacy.js [hunk 3]
```diff
@@ -14,7 +15,7 @@ export {recordEvent, isAnalyticsAllowed, setIsAnalyticsAllowed}
  * @param {string} component
  * @param {string} name
  * @param {string} defaultValue
- * @return {boolean} value of the setting
+ * @return {string} value of the setting
```
Comment (pablo-mayrgundter) [inline src/privacy/Privacy.js:18]: Good catch.

**Review by pablo-mayrgundter — CHANGES_REQUESTED** [review]
All looks good, but for me to run with node18 needed to remove bldrs/package.json#canvas.  Also removed gl.  See how goes for you

File: package.json (hunk 1 above)
Comment (oogali) [reply package.json:75]: Without this, Jest expects the module to be already transformed into a CommonJS artifact. Since the packages listed in this regex are ES6 modules, Jest errors out on the ES6 import/export syntax.

This directive tells Jest to transpile these matching modules prior to executing the tests.

https://jestjs.io/docs/configuration#transformignorepatterns-arraystring

**Comment by OlegMoshkovich** [comment]
@oogali @pablo-mayrgundter 

Is this ready to merge?

**Comment by pablo-mayrgundter** [comment]
Hi O, could you take another pass at this or revert?
